Source/LibationUiBase/GridView/GridEntry[TStatus].cs

```diff
@@ -154,9 +154,13 @@ private void UserDefinedItem_ItemChanged(object sender, string itemName)
 			if (udi.Book.AudibleProductId != Book.AudibleProductId)
 				return;
 
-			//If UserDefinedItem was changed on a different Book instance (such as when batch liberating via menus),
-			//EntryStatu's Book instance will not have the current DB state.
-			Liberate.Book = udi.Book;
+			if (udi.Book != LibraryBook.Book)
```

Comment:
Did we not correctly update the library book elsewhere? It feels very weird for it to be a contingent UI responsibility like this.

Reply:
It's not really about correctly/incorrectly updating LibraryBook, and this isn't the UI.  Think MVVM. GridEntry is the ViewModel. It's been notified (via UserDefinedItem.ItemChanged) that the model has changed and is updating itself.  It then notifies the view (the actual UI) that it needs to change.

I could get into why this wasn't necessary prior to my refactor of GridEntry into LibationUiBase if you'd like, but the short version is that it was **not** better. It still had the same bug, but it was covered up by a sloppy way in which the view was updated.